tests/unittests/TestMain.cpp

```diff
@@ -15,10 +15,12 @@
  */
 
 #include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Signals.h"
 #include "gtest/gtest.h"
 
 int main(int argc, char **argv) {
   ::testing::InitGoogleTest(&argc, argv);
+  llvm::sys::PrintStackTraceOnErrorSignal(llvm::StringRef());
```

Comment:
Should it pass `argv[0]` as the argument, because it is the name of the current binary?
See e.g. https://github.com/llvm-mirror/clang-tools-extra/blob/master/clang-tidy/tool/ClangTidyMain.cpp for an example of calling `PrintStackTraceOnErrorSignal`

Reply:
I dunno.  The docs say it's used to find the symbolizer relative to the current binary (http://llvm.org/doxygen/namespacellvm_1_1sys.html#ab062fd190912d9ca714311df7cbe51d5), which seems useful for llvm tools but maybe not for Glow?  I guess it can't hurt.